internal/build/container_ops_test.go

```diff
@@ -231,7 +231,7 @@ lrwxrwxrwx    1 123      456 (.*) fake-app-symlink -> fake-app-file
 
 			h.AssertEq(t, errBuf.String(), "")
 			if osType == "windows" {
-				h.AssertContains(t, outBuf.String(), `01/01/1980  12:00 AM                69 ...                    stack.toml`)
+				h.AssertContainsMatch(t, strings.ReplaceAll(outBuf.String(), "\r", ""), `(.*) 69 (.*) stack.toml`)
```

Comment:
Is there any way to keep the date here? It's useful to validate that we strip that part of the metadata. If it was failiing, I suspect you may need to set your Windows machine to UTC: `Set-TimeZone -Id UTC`

Reply:
> Is there any way to keep the date here? It's useful to validate that we strip that part of the metadata. If it was failiing, I suspect you may need to set your Windows machine to UTC: `Set-TimeZone -Id UTC`

Good point @micahyoung. I've added back the date and setting the timezone on my local VM worked.